core/src/main/java/com/rettichlp/unicacityaddon/listener/faction/state/WantedListener.java

```diff
@@ -129,6 +127,16 @@ public void onChatReceive(ChatReceiveEvent e) {
                         .createComponent());
```

Comment:
Probier mal ob das auch funktioniert bitte :)

```suggestion
                        .of(isAddonPlayer ? "[⬆]" : "").color(ColorCode.BLUE)
                                .hoverEvent(HoverEvent.Action.SHOW_TEXT, Message.getBuilder().of("Aktivität eintragen").color(ColorCode.DARK_BLUE).advance().createComponent())
                                .clickEvent(ClickEvent.Action.RUN_COMMAND, "/activitytest kills")
                                .advance()
                        .createComponent());
```